### PR TITLE
Moving dependency from .classpath and libs directory to maven pom.xml

### LINKED ISSUE #388: Support for running test cases from Intellij Idea IDE
Intellij Idea IDE is unable to run junits, because the org.openjdk.jmh dependency is not available in the classpath. This dependency is available in eclipse, as part of the .classpath file. It can be declared as a dependency in maven pom.xml, so that it is available on all IDE's. 

For instance, when the test class org.numenta.nupic.network.LayerTest is executed from Intellij Idea, we see the following error, indicating that classes from jmh library, were not found in the classpath.

screen shot 2016-01-31 at 5 12 59 pm

A fix is available as part of the PR #387 

Cc. @cogmission

### PR DESCRIPTION
**Pull request by mandarx**
Fixes #388 
Making this change, to make org.numenta.nupic.network.LayerTest, run on any IDE. LayerTest was failing on Intellij Idea, due to missing dependency. Now, we can potentially clean up jmh-core dependency from .classpath file and libs directory.

### CODE REVIEW

**Comment by cogmission** [comment]
Hi @mandarx,

Welcome! Thank you for contributing to htm.java! There are a few "rules" or steps that all contributions must adhere to. 

- Please fill out the contributor's license: http://numenta.org/licenses/cl/

- All PR's (Pull Request) must first be associated with a github issue describing what needs to be done. As an example see [this PR:](https://github.com/numenta/htm.java/pull/386)  The words "Fixes #nnn" must be in the first comment...

- The CI (continuous integration) must pass, which yours does....

- The coverage report (the amount of test coverage) must remain the same or increase (yours remains the same, so it is ok).

So you've done the last two, but you must adhere to the first two rules by filling out the CL, and creating an issue and quoting it in the first comment of your PR.

Thanks for helping make htm.java a better project!

Cheers,
David

**Comment by mandarx** [comment]
Hi @cogmission , 

Thanks for the guidance! I have followed accordingly, on the first two rules.

Regards,
Mandar

**Comment by cogmission** [comment]
@mandarx,

I'm not really that experienced using maven, but I was wondering why you omitted the "version" attribute from the dependency section you added? Isn't that necessary? Just wondering...

**Comment by mandarx** [comment]
Hi @cogmission ,

You are right.
I am updating the artifact with the latest available version(1.10.3).

Thanks,
Mandar

**Comment by cogmission** [comment]
> Coverage decreased (-0.0%)

@mandarx Coveralls is a little retarded sometimes that isn't it. Go ahead and revert it once more back to 1.11.3.

I will add a test to your PR so this will pass...

**Comment by cogmission** [comment]
> remote: Permission to mandarx/htm.java.git denied to cogmission.
@mandarx please give me push permissions on mandarin/htm.java, I will create a PR which you should then merge, which then will contain a new test. Following that, you can then "re-push" up to your PR with the new test content and the PR's test should pass? Please let me know if you need help with this?

**Comment by mandarx** [comment]
Hi @cogmission , Just added you as a collaborator, to the repo: https://github.com/mandarx/htm.java.git

**Comment by cogmission** [comment]
Hi @mandarx I pushed it directly in through your PR, so let's see if the coverage examination succeeds (it should)...

**Comment by cogmission** [comment]
Ok there we go! Merging now...

**Comment by mandarx** [comment]
Thanks @cogmission !

**Comment by cogmission** [comment]
Your welcome! Thank you.

File: .project [hunk 1]
```diff
@@ -1,18 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>htm.java</name>
+	<name>mandarx</name>
```
Comment (antidata) [inline .project]: @cogmission Why it was changed? :)

Comment (cogmission) [inline .project]: Oops! Thanks @antidata ! :+1:

**Pull request merged**